networking/networking.go

```diff
@@ -50,9 +51,20 @@ type Networking struct {
 	nets   []activeNet
 }
 
+// NetConf local struct extends plugin.NetConf with information about masquerading
+// similar to CNI plugins
+type NetConf struct {
+	plugin.NetConf
+	IPMasq bool `json:"ipMasq"`
+}
+
 // Setup creates a new networking namespace and executes network plugins to
 // setup private networking. It returns in the new pod namespace
-func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, privateNetList common.PrivateNetList, localConfig string) (*Networking, error) {
+func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, privateNetList common.PrivateNetList, localConfig, flavor string) (*Networking, error) {
+	if flavor == "kvm" {
```

Comment:
right, it should be a type alias eg. `type flavorType string` but I don't want mess that with this PR because it's unrelated change - I've added it as a todo bullet point in follow up PR